app/models/question_format.rb

```diff
@@ -12,15 +12,25 @@
 #
 
 class QuestionFormat < ActiveRecord::Base
+  include ValidationMessages
+  include ValidationValues
 
+  ##
+  #
+  FORMAT_TYPES = %i[textarea textfield radiobuttons checkbox dropdown
+                    multiselectbox date rda_metadata]
   ##
   # Associations
   has_many :questions
 
-  enum formattype: [ :textarea, :textfield, :radiobuttons, :checkbox, :dropdown, :multiselectbox, :date, :rda_metadata ]
+  enum formattype: FORMAT_TYPES
+
+  validates :title, presence: { message: PRESENCE_MESSAGE },
+                    uniqueness: { message: UNIQUENESS_MESSAGE }
 
-  validates :title, presence: {message: _("can't be blank")}, uniqueness: {message: _("must be unique")}
+  validates :description, presence: { message: PRESENCE_MESSAGE }
```

Comment:
description can be nil. Not sure if the field is even necessary.

Reply:
@xsrust what do you think? can we remove the column?

Reply:
Description can be nil, but in the case of the RDA question type, we need the field to hold the URL of the API.
We could change the name of the field on the db though, as 'description' isnt that descriptive of what we're using it for.

Reply:
Added new issue for that